source/includes/php/useElement.md

```diff
@@ -0,0 +1,302 @@
+# Manage a single Element
+
+## Original values
+
+> Get original values
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+$product = \Shoprunback\Elements\Product::retrieve('1f27f9d9-3b5c-4152-98b7-760f56967dea');
+
+echo $product->label; // Prints: 'My label'
+
+$product->label = 'New label';
+
+echo $product->label; // Prints: 'New label'
+
+echo $product->_origValues->label; // Prints 'My label'
+
+$product->refresh();
+
+echo $product->label; // Prints 'My label'
+```
+
+You can get the original values of an Element whenever you want by getting the `_origValues` attribute.
+
+**Never edit the** `_origValues`, there are a landmark.
```

Comment:
Nonsense.

What do you mean with landmark?

If you have trouble finding words in english, do not google translate them as it sucks. Write them in french with a TODO TRANSLATE next to it. The rest of the team can help